test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1232,52 +1232,56 @@ def test_cgroup_cpuset(self):
             self.skipTest('Test requires at least two physical CPUs')
         name = 'CGROUP4'
         self.load_config(self.cfg3 % ('', '', '', self.swapctl, ''))
+        # Submit two jobs
         a = {'Resource_List.select': '1:ncpus=1:mem=300mb:host=%s' %
              self.hostA, ATTR_N: name + 'a'}
         j1 = Job(TEST_USER, attrs=a)
         j1.create_script(self.cpuset_mem_script)
         jid1 = self.server.submit(j1)
-        a = {'job_state': 'R'}
-        self.server.expect(JOB, a, jid1)
-        attrib = [ATTR_o, 'exec_host']
-        self.server.status(JOB, attrib, jid1)
-        filename1 = j1.attributes[ATTR_o]
-        self.tempfile.append(filename1)
-        ehost1 = j1.attributes['exec_host']
         b = {'Resource_List.select': '1:ncpus=1:mem=300mb:host=%s' %
              self.hostA, ATTR_N: name + 'b'}
         j2 = Job(TEST_USER, attrs=b)
         j2.create_script(self.cpuset_mem_script)
         jid2 = self.server.submit(j2)
         a = {'job_state': 'R'}
+        # Make sure they are both running
+        self.server.expect(JOB, a, jid1)
         self.server.expect(JOB, a, jid2)
+        # Status the jobs for their output files
+        attrib = [ATTR_o, 'exec_host']
```

Comment:
Do you still need exec_host?

Reply:
I think we still need it because it is used in the self.server.status() calls that follow.

Reply:
I don't see exec_host used; since you put the host in the select line, you (correctly) assume that the host will be hostA.

Reply:
Ahh... I think I see your point now. We're asking for the status of two attributes, when we really only need ATTR_o. I'll remove the 'exec_host' attribute from the list.